pkg/security/serializers/deserializers.go

```diff
@@ -0,0 +1,151 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+
+package serializers
+
+import (
+	json "encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/security/secl/model"
+	"github.com/DataDog/datadog-agent/pkg/security/utils"
+)
+
+func getPointerValue[T uint32 | uint64 | bool](p *T) T {
+	if p != nil {
+		return *p
+	}
+	var def T
+	return def
+}
+
+func getInnerTime(et *utils.EasyjsonTime) time.Time {
```

Comment:
This could be a method on `EasyjsonTime` struct directly. This would mean that you don't need to make `Inner` public

Reply:
Suggestion applied, thanks!